scripts/README.md

```diff
@@ -0,0 +1,13 @@
+# Scripts
+
+## Push Scripts
+
+The push scripts in this directory will build an image locally and push it to OpenShift's registry.
+They should only be used if remote builds in OpenShift are failing. 
```

Comment:
An idea: There's probably a way to combine all these scripts into one, passing the project name at the command line. Something like this? Syntax may be a bit off.

```
REG_BASE = "docker-registry.webplatformsunpublished.umich.edu"
oc project "$1"
docker login -u `oc whoami` -p `oc whoami -t` "$REG_BASE"
docker build -t  "$REG_BASE"/"$1"/officehours .
docker push  "$REG_BASE"/"$1"/officehours
```